Makefile

```diff
@@ -11,6 +11,12 @@ REPORTS = $(ROOT_DIR)/reports
 
 SKIPPER_PARAMS ?= -i
 
+# Openshift CI params
+OPENSHIFT_CI := $(or ${OPENSHIFT_CI}, "false")
+JOB_TYPE := ${or ${JOB_TYPE}, ""}
```

Comment:
My understanding is because these are environment variables and not Makefile variables I'm trying to pull in.

The syntax on these things are not super clear, and depending on the version of Make it will be present as both environment variable and Makefile variable.. I believe.

Reply:
My understanding is because I'm trying to pull in environment variables and not Makefile variables.

```
export OPENSHIFT_CI=true
make bring_assisted_service
```
or

```
make bring_assisted_service OPENSHIFT_CI=true
```

In most modern versions of make these will probably have the same effect, because it will pull in environment variables as Makefile variables. In Prow it is setting the environment variable though.. so that's why I was trying to pull it in that way.

Reply:
Also looking at it.. the outer `{...}` are a typo and the inner `${JOB_TYPE}` are to select the environment variable in this line. Good catch.